src/azure-cli-core/azure/cli/core/parser.py

```diff
@@ -400,46 +609,50 @@ def _check_value(self, action, value):  # pylint: disable=too-many-statements, t
                                 import subprocess
                                 import platform
                                 exit_code = subprocess.call(cmd_list, shell=platform.system() == 'Windows')
-                                telemetry.set_user_fault("Extension {} dynamically installed and commands will be "
-                                                         "rerun automatically.".format(ext_name))
```

Comment:
@fengzhou-msft We don't need set userfault here, right?

Reply:
We need this message in telemetry to get the numbers for dynamic install with automatic rerun.
Here actually the same command will run twice, I set the first command to be UserFault with the message, the second command which is rerun will have normal result.

I'm OK to not set user fault for the first command, but you need to put the message in resultSummary to distinguish it.

Reply:
will set userfault

Reply:
Modified.